Makefile

```
@@ -19,84 +19,96 @@ else
 	CGO_ENABLED=1 GOOS=darwin GOARCH=arm64 \
 		CC="clang -arch arm64" \
 		CGO_CFLAGS="-arch arm64" CGO_LDFLAGS="-arch arm64" \
-		go build -ldflags="$(LDFLAGS)" -o dist/bin/cem-darwin-arm64 .
+		go build -ldflags="$(LDFLAGS)" -o $@ .
 
-# Windows targets for go-release-workflows (use shared Containerfile.windows)
+# Windows targets (use shared Containerfile.windows via podman)
 SHARED_WINDOWS_CC_IMAGE := cem-shared-windows-cc
 
+.PHONY: build-windows-cc-image rebuild-windows-cc-image build-shared-windows-image
+build-windows-cc-image:
```

checkmake (0.3.2)

[warning] Target body for "build-windows-cc-image" exceeds allowed length of 5 lines (6).

(maxbodylength)